app/features/Kaichat/core.py

```diff
@@ -40,7 +40,7 @@ def executor(user_name: str, user_query: str, messages: list[Message], k=10):
 
     prompt = build_prompt()
     
-    llm = GoogleGenerativeAI(model="gemini-1.0-pro") 
+    llm = VertexAI(model_name="gemini-1.0-pro") 

```

Comment:
Why is `langchain_google_genai.GoogleGenerativeAI` being changed to `langchain_google_vertexai.VertexAI` in another feature, while you're still using `langchain_google_genai.GoogleGenerativeAI` in your syllabus generator?